### PR TITLE
a11y icons for versions

### PR DESCRIPTION
**Pull request by brianteeman**
When reading our default markup for rendering icons, assisistive technology may have the following problems.

The assistive technology will not find any content to read out to a user
The assistive technology will read the unicode equivalent, which does not match up to what the icon means in context, or worse is just plain confusing. In our use case it is always plain wrong. For example the unicode character used to display the trashed icon is \4c which is equal to L

When an icon is not an interactive element the simplest way to provide a text alternative is to use the aria-hidden="true" attribute on the icon and to include the text with an additional element, such as a < span>, with appropriate CSS to visually hide the element while keeping it accessible to assistive technologies.

In this case we have the text and it is displayed so we just need to add the aria-hidden to prevent the icon being "read aloud"

This PR addresses the use case shown in the images below

screenshotr15-46-21
screenshotr15-46-30
screenshotr15-48-05

### CODE REVIEW

**Review by nibra — APPROVED** [review]
Good finding!

**Review by dgrammatiko — APPROVED** [review]

**Comment by Quy** [comment]
Missed this one `<span class="icon-featured"></span>&nbsp;`.

File: administrator/components/com_contenthistory/views/history/tmpl/modal.php [hunk 1]
```diff
@@ -150,7 +150,7 @@
 							<?php echo JHtml::_('date', $item->save_date, 'Y-m-d H:i:s'); ?>
 						</a>
 						<?php if ($item->sha1_hash == $hash) : ?>
-							<span class="icon-featured"></span>&nbsp;
+							<span class="icon-featured" aria-hidden="true"><div class="element-invisible"><?php echo JText::_('JFEATURED'); ?></div></span>&nbsp;
```
Comment (Quy) [inline administrator/components/com_contenthistory/views/history/tmpl/modal.php]: Use `<span>` instead as coded elsewhere: 
`<span class="icon-publish hasTooltip" aria-hidden="true" title="<?php echo JText::_('COM_BANNERS_COUNT_PUBLISHED_ITEMS'); ?>"><span class="element-invisible"><?php echo JText::_('COM_BANNERS_COUNT_PUBLISHED_ITEMS'); ?></span></span>`

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 2f1b80da703ce752470f81fc98b4614d5e1d0c86<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14698">issues.joomla.org/tracker/joomla-cms/14698</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 2f1b80da703ce752470f81fc98b4614d5e1d0c86<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14698">issues.joomla.org/tracker/joomla-cms/14698</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14698">issues.joomla.org/tracker/joomla-cms/14698</a>.</sub>

**Comment by brianteeman** [comment]
thanks

**Pull request merged**